roles/infra/templates/proxy.j2

```diff
@@ -0,0 +1,13 @@
+{{ ansible_managed | comment }}
+
+server {
+  listen 80 default_server;
+  server_name {{ inventory_hostname }};
+  
+  location / {
+    # Allowing for a `burst` of up to 5 requests beyond the specified rate limit. The `nodelay` parameter ensures that excessive requests beyond the burst limit are immediately rejected with a 429 error response instead of being queued.
+    limit_req zone=limited burst=5 nodelay;
```

Comment:
This comment describes what the options do, which can easily be found in the documentation, but does not explain _why_ we set up `burst`. And I still have not understood why we do it 🙂

Reply:
A good explanation of why it is recommended https://www.nginx.com/blog/rate-limiting-nginx/